### PR TITLE
Convert to radium

### PR DESCRIPTION
**Pull request by dhoward**
npm run test

There are comments in the test files to document the logic, I think I covered all the use cases currently in the codebase. but let me know if I missed anything.

### CODE REVIEW

File: test/convert-to-radium-test.output.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <span className="icon-class"></span>
+                <span className="icon-class"></span>
+                <span style={{width: "10px"}} className="icon-class"></span>
+
+                // js objects will get converted to style tags
+                // and merged with Object.assign if necessary.
+                // Calls to the classNames module will be stripped out
+                <span style={styles.one}></span>
+                <span style={styles.one}></span>
+                <span style={Object.assign({}, styles.one, styles.two)}></span>
```
Comment (tmbtech) [inline test/convert-to-radium-test.output.js]: instead of using `Object.assing({}, foo, bar)` is there a way to do `{{...foo, ...bar}}`

Comment (dhoward) [inline test/convert-to-radium-test.output.js]: Pretty sure there is, I'll check it out

**Comment by doctyper** [comment]
@dhoward Worst case, you can add the radium import and it will warn the developer that there is an unused radium import.

**Comment by dhoward** [comment]
@doctyper got it adding the decorator, just having it treat it as a generic expression and removing the `;` that i can't get it to not add

**Comment by doctyper** [comment]
:+1: It also needs to have:

``` js
import radium from "react-wildcat-radium";
```

**Comment by dhoward** [comment]
Oops, forgot that. In there now.

**Comment by doctyper** [comment]
@dhoward There's one use-case still missing:

> If your styles object has dynamic styles (hover, media queries), add a key="" attribute

Sounds tricky. Worst-case, I'd be happy with:

> If the element has a `className={styles.foo}` and does not have a `key`, then set `key="foo"`

Doable?

**Comment by dhoward** [comment]
@doctyper Second one is definitely doable, I'll take a shot at checking for interactive styles and fall back to that if I have to.

**Comment by doctyper** [comment]
Here are the relevant Radium docs:
https://github.com/FormidableLabs/radium/tree/master/docs/guides#styling-multiple-elements-in-a-single-component

**Pull request merged**